rpcs3/Loader/TRP.cpp

```diff
@@ -145,7 +145,7 @@ void TRPLoader::RenameEntry(const char *oldname, const char *newname)
 	{
 		if (!strcmp(entry.name, oldname))
 		{
-			memcpy((void*)entry.name, newname, 32);
+                        memcpy((void*)entry.name, newname, std::min((int)strlen(newname), 32));
```

Comment:
how many spaces are you gonna add in the next commit?

Reply:
Use strcpy_trunc

Reply:
123